public/components/health-check/container/health-check.container.tsx

```diff
@@ -67,39 +65,26 @@ const checks = {
     awaitFor: ["api"],
   },
   pattern: {
-    title: 'Check alerts index pattern',
-    label: 'Alerts index pattern',
+    title: 'Check index pattern',
+    label: 'Index pattern',
```

Comment:
The `alert` word helps to identify that the check is related to that index pattern. I didn't remove it.

Reply:
Added Alerts in title and label